### PR TITLE
Changelog for 11.2.0

### PR DESCRIPTION
**Pull request by lilgreenbird**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,21 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 
+## [11.2.0] Preview Release
```
Comment (tkyc) [inline CHANGELOG.md:6]: Needs to be `Stable Release`

Comment (David-Engel) [inline CHANGELOG.md:6]:
```suggestion
## [11.2.0] Stable Release
```

**Comment by tkyc** [comment]
Also make sure the pom, gradle and license files are updated with the appropriate version. If so ignore this comment.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,21 @@ All notable changes to this project will be documented in this file.
...
+## [11.2.0] Stable Release
+### Added
+- Added support for caching parameter metadata for Always Encrypted with secure enclaves [1866](https://github.com/microsoft/mssql-jdbc/pull/1866)
+- Added explicit dependency for com.microsoft.azure.msal4j (was a transitive dependency in previous releases) [1863](https://github.com/microsoft/mssql-jdbc/pull/1863)
+
+### Changed
+- Updated dependency versions of com.azure.azure-security-keyvault-keys and com.azure.azure-identity [1863](https://github.com/microsoft/mssql-jdbc/pull/1863)
+- Send TDS version 8 in Login7 when in strict mode (encrypt=strict) [1870](https://github.com/microsoft/mssql-jdbc/pull/1870)
+### Fixed issues
+- Fixed race condition with addressList which may result in IndexOutOfBoundsException when establishing multiple conntions [1855](https://github.com/microsoft/mssql-jdbc/pull/1855)
```
Comment (Jeffery-Wasty) [inline CHANGELOG.md:15]:
```suggestion
- Fixed race condition with addressList which may result in IndexOutOfBoundsException when establishing multiple connections [1855](https://github.com/microsoft/mssql-jdbc/pull/1855)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,21 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 
+## [11.2.0] Stable Release
+### Added
+- Added support for caching parameter metadata for Always Encrypted with secure enclaves [1866](https://github.com/microsoft/mssql-jdbc/pull/1866)
+- Added explicit dependency for com.microsoft.azure.msal4j (was a transitive dependency in previous releases) [1863](https://github.com/microsoft/mssql-jdbc/pull/1863)
+
```
Comment (Jeffery-Wasty) [inline CHANGELOG.md:10]: Remove empty line to be consistent with previous updates

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,21 @@ All notable changes to this project will be documented in this file.
...
+### Changed
+- Updated dependency versions of com.azure.azure-security-keyvault-keys and com.azure.azure-identity [1863](https://github.com/microsoft/mssql-jdbc/pull/1863)
+- Send TDS version 8 in Login7 when in strict mode (encrypt=strict) [1870](https://github.com/microsoft/mssql-jdbc/pull/1870)
+### Fixed issues
+- Fixed race condition with addressList which may result in IndexOutOfBoundsException when establishing multiple connections [1855](https://github.com/microsoft/mssql-jdbc/pull/1855)
+- Validate that the serverName field of the connection string does not have an equal sign [1853](https://github.com/microsoft/mssql-jdbc/pull/1853)
```
Comment (David-Engel) [inline CHANGELOG.md:16]: Move this item to the Changed section

**Review by David-Engel — APPROVED** [review]

**Review by Jeffery-Wasty — APPROVED** [review]

**Pull request merged**